talisker/gunicorn.py

```diff
@@ -58,16 +58,8 @@ def prometheus_multiprocess_worker_exit(server, worker):
         multiprocess.mark_process_dead(worker.pid)
 
 
-class GunicornLogger(gstatsd.Statsd):
-    """Custom gunicorn logger to use structured logging.
-
-    Based on the statsd gunicorn logger, and also increases timestamp
-    resolution to include msec in access and error logs.
-    """
-    def __init__(self, cfg):
-        super(GunicornLogger, self).__init__(cfg)
```

Comment:
Yes, we the only part we were still using was the log metrics, which I moved the our class.